src/routes/businesses.ts

```diff
@@ -0,0 +1,289 @@
+import _ from "lodash";
+import moment from "moment";
+import HttpStatusCodes from "http-status";
+
+import {
+  saveBusiness,
+  redlock,
+  setBusinessOrigin,
+  saveAccountToBusinessId,
+  removeBusiness,
+  getBusiness,
+  findBusinesses,
+} from "../db";
+
+import generateID from "../helpers/id";
+import { storeBusinessInSortedSet } from "../helpers/businesses";
+import { BusinessWebhookEvent, MockBusiness } from "../helpers/types";
+import {
+  createBusinessChangeRequest,
+  createChangeRequest,
+} from "./changeRequest";
+import { triggerWebhook } from "../helpers/webhooks";
+
+const ERROR_NOT_FOUND_ID = "0a5ec2ea-6772-11e9-a656-02420a868404";
+
+export const createBusiness = async (req, res) => {
+  const businessId = generateID(); // Do not exceed 36 characters
+  let createdBusiness;
+  const businessLockKey = `redlock:${process.env.MOCKSOLARIS_REDIS_PREFIX}:business:${businessId}`;
+
+  await redlock.using([businessLockKey], 5000, async (signal) => {
+    if (signal.aborted) {
+      throw signal.error;
+    }
+    const createdAt = moment();
+
+    const business = {
+      ...req.body,
+      id: businessId,
+      createdAt: createdAt.toISOString(),
+    };
+
+    createdBusiness = await saveBusiness(business).then(() => {
+      res.status(200).send({
+        id: businessId,
+        ...req.body,
+      });
+    });
+
+    await storeBusinessInSortedSet(business);
+
+    if (business.account?.id) {
+      await saveAccountToBusinessId(business.account, businessId);
+    }
+
+    if (req.headers.origin) {
+      await setBusinessOrigin(businessId, req.headers.origin);
+    }
+  });
+
+  return createdBusiness;
+};
+
+export const deleteBusiness = async (req, res) => {
```

Comment:
let's remove everything related to 'deleteBusiness'. Solaris doesn't allow it and it is not used in mocksolaris ui